test/rules/no-top-level-hooks.js

```diff
@@ -22,7 +22,24 @@ ruleTester.run('no-top-level-hooks', rules['no-top-level-hooks'], {
         'beforeEach.foo()',
         'afterEach.foo()',
         'var before = 2; before + 3;'
-    ],
+    ].reduce(function (memo, test) {
+        memo.push(test);
+        memo.push({
+            code: test,
```

Comment:
I just realized that the test isn't correct implement, because the code doesn't contain `parallel`. Will fix it tomorrow